lib/cucumber/cli/configuration.js

```diff
@@ -18,8 +18,25 @@ function Configuration(options, args) {
     });
   }
 
-  var expandedFeaturePaths = Cucumber.Cli.FeaturePathExpander.expandPaths(unexpandedFeaturePaths);
-
+  var expandedFeaturePaths;
+  try {
+    expandedFeaturePaths = Cucumber.Cli.FeaturePathExpander.expandPaths(unexpandedFeaturePaths);
+  } catch (e) {
+      if (e.code === 'ENOENT' || e.code === 'ENOTDIR') {  
+        var errorMessage = 'No such file or directory - ';
+        var basePath = path.basename(e.path);
```

Comment:
I recall thinking about this and I chose the base name for a reason but I wrote this a week ago so I forget why. Let me go back and have a look.